crates/sui-framework-tests/src/metered_verifier.rs

```diff
@@ -200,9 +203,11 @@ fn test_metered_move_bytecode_verifier() {
     let package = build(&path).unwrap();
     packages.push(package.get_dependency_sorted_modules(with_unpublished_deps));
 
+    let signing_config = VerifierSigningConfig::default();
```

Comment:
nit (optional): This seems to be one big file, so you could probably get away with creating this config once?

Reply:
Sadly a bunch of tests all in this file. So probably best to keep as-is I think.